backend/store/migration/dev/LATEST.sql

```diff
@@ -637,12 +637,9 @@ CREATE TABLE pipeline (
     created_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
     updater_id INTEGER NOT NULL REFERENCES principal (id),
     updated_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
-    name TEXT NOT NULL,
-    status TEXT NOT NULL CHECK (status IN ('OPEN', 'DONE', 'CANCELED'))
```

Comment:
What are the rules for calculating the pipeline status from task status?
 
Say, a pipeline contains 3 tasks. The first 2 completes successfully. The last 1 is canceled. In such case, should it be marked as done or canceled?

Reply:
The pipeline status is no longer used. This field was used to get active pipeline in the task scheduler which is already optimized.

Reply:
> We will calculate the pipeline status by tasks.

Then, the PR description should be altered

Reply:
Done.